custom_static/js/handleFormSubmit.js

```diff
@@ -0,0 +1,10 @@
+function handleFormSubmit(event) {
+  event.preventDefault(); // Prevent the form
```

Comment:
Could you remove this and implement a Contact Form model to actually store the submitted forms?